Traceback (most recent call last):
  File "/home/mateus/Documentos/pyhton/excecao.py", line 4, in <module>
    print(1/0)
ZeroDivisionError: division by zero
Traceback (most recent call last):
  File "/home/mateus/Documentos/pyhton/excecao.py", line 4, in <module>
    print(1/0)
ZeroDivisionError: division by zero
Traceback (most recent call last):
  File "/home/mateus/Documentos/pyhton/excecao.py", line 4, in <module>
    print(1/0)
ZeroDivisionError: division by zero
Traceback (most recent call last):
  File "/home/mateus/Documentos/pyhton/excecao.py", line 4, in <module>
    print(1/0)
ZeroDivisionError: division by zero
